CGFloatType/CGFloatType.h

```diff
@@ -37,6 +37,9 @@ CGFloat CGFloatWithDouble(double value);
 /// Returns a CGFloat from a float
 CGFloat CGFloatWithFloat(float value);
 
+#pragma mark - CGFloat comparison
+
+extern BOOL CGFloatEquals(CGFloat x, CGFloat y);
```

Comment:
Hmm, your right. I didn't see those ones from this PR. Just stood out because the others shown in PR didn't have it.